src/styles.scss

```diff
@@ -7,6 +7,7 @@
 @import "./app/style-components/pricing-table.scss";
 @import "./app/style-components/teaser.scss";
 @import "./app/style-components/button.scss";
+@import "./app/components/speakers/speakers.component.scss";
```

Comment:
this shouldn't be here as its just should be part of the speakers component itself.